$OpenBSD$

replace hid_init -> hidapi_hid_init
remove hid_write_control which isn't available

Index: rpcs3/ds4_pad_handler.cpp
--- rpcs3/ds4_pad_handler.cpp.orig
+++ rpcs3/ds4_pad_handler.cpp
@@ -705,7 +705,6 @@ int ds4_pad_handler::SendVibrateData(const std::shared
 		outputBuf[76] = (crcCalc >> 16) & 0xFF;
 		outputBuf[77] = (crcCalc >> 24) & 0xFF;
 
-		return hid_write_control(device->hidDevice, outputBuf.data(), DS4_OUTPUT_REPORT_0x11_SIZE);
 	}
 	else
 	{
@@ -728,7 +727,7 @@ bool ds4_pad_handler::Init()
 	if (is_init)
 		return true;
 
-	const int res = hid_init();
+	const int res = hidapi_hid_init();
 	if (res != 0)
 		fmt::throw_exception("hidapi-init error.threadproc");
 
